### ISSUE TITLE
Help with NUTEditor

### ISSUE THREAD

**Issue by tsulocalize** [marcussacana/SacanaWrapper#17]
Hi, I'm trying to open .nut files of totono with your tools.

I compiled sacanawrapper and installed NUTEditor as seen below.

![20211121_192704](https://user-images.githubusercontent.com/94783621/142758276-61754ca7-089b-4388-99d5-d07b207d3b14.png)

However, when i drag&drop my .nut files on stringtool.exe nothing happened.

stringtool.exe is on "SacanaWrapper-master\StringTool\bin\Release". Is it right place?

Could you give any advice?

**Comment by marcussacana** [comment]
Install only one of them... some nut files works with the first plugin, someothers works with the second plugin

**Comment by tsulocalize** [comment]
Thanks.

I tried two plugins one by one, but it still doesn't work.

![20211122_013302](https://user-images.githubusercontent.com/94783621/142770705-a841b65c-56ef-405f-aa33-b1da553656d1.png)

When I put the .nut file on stringtool.exe, the console is closed very quickly and nothing changed.

I don't know what the problem is...
------------------------------------------------------------
forget it. I managed to make it. Thank you!
